extensions/wikia/Recirculation/js/helpers/sponsoredContent.js

```diff
@@ -38,9 +39,28 @@ define('ext.wikia.recirculation.helpers.sponsoredContent', [
 	}
 
 	function getApplicableContent(sponsoredContent) {
-		return sponsoredContent.filter(function (el) {
-			return !el.geos.length || el.geos.indexOf(userGeo) !== -1;
+		var geoSpecificContent = sponsoredContent.filter(function (el) {
```

Comment:
I'm still confused with this logic. We have "geoSpecific" items here which are filtered by both geo and wikiId.